DESCRIPTION

```diff
@@ -19,11 +23,30 @@ Encoding: UTF-8
 LazyData: true
 biocViews: Software, Infrastructure
 Imports:
-    Biostrings
+    Biostrings,
+    utils,
+    stats,
+    xlsx,
+    readxl,
+    visNetwork,
+    dplyr,
+    biomaRt,
+    ggplot2,
+    biogridr,
+    devtools
 Suggests:
     testthat,
     BiocStyle,
     knitr,
     rmarkdown
+Remotes: NElnour/biogridr
 RoxygenNote: 6.1.1
 VignetteBuilder: knitr
+Collate: 
```

Comment:
Please check where the following changes come from. I am not sure what "Collate" denotes for, but all of the files besides "GGI_PPI_Integrator.R" should not be shown as altered.

Reply:
Collation describes the order with which the different files need to be built for my tool to work. I don't think it indicates that these files are necessarily altered.

Reply:
Requiring collation is often a sign of a brittle code architecture. Can you remove the need to load in a particular sequence, or elaborate why this is necessary? Moreover, a `Collate:` statement in the `DESCRIPTION` affects all co-authors...